exercises/yacht/.meta/template.j2

```diff
@@ -0,0 +1,11 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual({{ case["property"] }}({{case["input"]["dice"]}}, {{ exercise }}.{{case["input"]["category"].upper().replace(' ', '_')}}), {{ case["expected"] }})
```

Comment:
There are built-in filters for the string transformation here. Using them is highly recommended:

```suggestion
        self.assertEqual(
            {{ case["property"] }}(
                {{ case["input"]["dice"]}},
                {{ exercise }}.{{ case["input"]["category"] | to_snake | upper }}
            ),
            {{ case["expected"] }}
        )
```